notebooks/official/sdk/SDK_Custom_Training_Python_Package_Managed_Text_Dataset_Tensorflow_Serving_Container.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>remove the uninstall for google-cloud-aiplatform, and add upgrade to the install version.</p><p> </p><p>Also missing USER_FLAG</p><p> </p><p>! pip3 install --upgrade google-cloud-aiplatform {USER_FLAG} -q</p><p>! pip3 install --upgrade tensorflow&nbsp;{USER_FLAG} -q</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/507/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>